etc/dnf/plugins/copr.conf

```diff
@@ -0,0 +1,6 @@
+[hub "fedora"]
```

Comment:
Let's dump it here in the diff .. I would simply prefer `[fedora]` ini header here:
- it is standard and native for configparser
- every other configuration in dnf/yum was fine with that so far, so I doubt we need to invent something new here (what `hub "fed" "ora"` means?, etc.)

Reply:
Please, add "Configuration" section to doc/copr.rst with description of config options and example of config file.
Btw, I'm also voting for simpler sections naming like [fedora]. It will be also consistent with configuration of repositories.